maintainers/maintainer-list.nix

```diff
@@ -1018,6 +1013,11 @@
     github = "ciil";
     name = "Simon Lackerbauer";
   };
+  cizra = {
+    email = "todurov+nix@gmail.com";
+    github = "cizra";
```

Comment:
As part of a brand new requirement can you please also add `githubid`. Your `githubid` is [2131991](https://api.github.com/users/cizra).